### DISCUSSION TITLE
Relocate all TRMM components in the same "program file" directory

### DISCUSSION THREAD

**Discussion by mbr-a** [amidaware/tacticalrmm#1570] [Q&A]
Is it a way to relocate all TRMM agent components (TRMM, Mesh, Chocolatey) in the same installation folder ?
I see that we can change default temp folder but not install folder.

Needs are / can be : 
- Change the default location name
- Reducing the "gas factory" effect between the different components used by TRMM

Thanks for your answers.

Regards,

**Comment by silversword411** [comment]
Not really
